reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md

```diff
@@ -21,17 +20,31 @@ Invoke-Expression [-Command] <String> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Invoke-Expression** cmdlet evaluates or runs a specified string as a command and returns the results of the expression or command.
-Without **Invoke-Expression**, a string submitted at the command line would be returned (echoed) unchanged.
+
+The `Invoke-Expression` cmdlet evaluates or runs a specified string as a command and returns the
+results of the expression or command. Without `Invoke-Expression`, a string submitted at the command
+line is returned (echoed) unchanged.
+
+Expressions are evaluated and run in the current scope. For more information, see [about_Scopes](../Microsoft.PowerShell.Core/About/about_Scopes.md).
 
 ## EXAMPLES
 
 ### Example 1: Evaluate an expression
+
+```powershell
+$Command = "Get-Process"
+$Command
+```
+
+```Output
+Get-Process PS
```

Comment:
Don't think the `PS` is supposed to be there.

Reply:
This still needs addressing.

Reply:
Oops. Fixed